app/views/govuk_publishing_components/components/_input.html.erb

```diff
@@ -7,7 +7,9 @@
   describedby ||= nil
   controls ||= nil
   data ||= nil
-  autocomplete ||= nil
+  disable_form_submit_check = true if disable_form_submit_check
+  reveal_password = type == "password" && reveal_password ? true : false
```

Comment:
I'd suggest removing the ternary operator and either replace it with a check for `true`:

```suggestion
  reveal_password = type == "password" && reveal_password == true
```

Or remove the ternary operator, which has the same result:
```suggestion
  reveal_password = type == "password" && reveal_password
```

Reply:
Thanks, have changed.